boards/shields/huatian_mt9m114/huatian_mt9m114.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	chosen {
+		zephyr,camera = &csi;
+	};
+};
+
+&lpi2c1 {
```

Comment:
Same comment as other shield overlay- we can't use the node labels that are specific to the board here, we need to define more generic labels that could be reused on other boards besides the RT1060 EVK

Reply:
If I remember well lpi2c1 is used across EVK RT1060, RT1064, etc.  for camera ?
But yes, we should be generic. I will use some alias so that the shield is as generic as possible.

Reply:
Yes, it very well may be- however, shields should _theoretically_ be able to work on a customer board, or another vendor's board provided they expose the same interfaces

Reply:
done